test/testSetup.ts

```diff
@@ -56,8 +57,28 @@ export class TestSetup {
     await this.checkForUncaughtErrors();
     try {
       await utilities.deleteScratchOrg(this.scratchOrgAliasName, this.reuseScratchOrg);
+      await this.deleteScratchOrgInfo();
     } catch (error) {
-      utilities.log(`Deleting scratch org failed with Error: ${(error as Error).message}`);
+      utilities.log(
+        `Deleting scratch org (or info) failed with Error: ${(error as Error).message}`
+      );
+    }
+  }
+
+  private async deleteScratchOrgInfo(): Promise<void> {
+    if (this.scratchOrgId) {
```

Comment:
How can we use `scratchOrgAliasName` instead of org Id?

Also, should this function be concerned about an org being reused?

Reply:
I could only find the orgId as something to identify the ScratchOrg in ScratchOrgInfo, so I think we can't use `scratchOrgAliasName`. Maybe I'm missing something?

For remote testing I don't think it actually matters as we always run all the tests with `reuseScratchOrg = false`. I was actually wondering if anyone ever sets it to true for local development and if not maybe we can get rid of all that code?

Reply:
@peternhale From what I've seen, it's always empty. I just checked again and it's always empty

Reply:
<img width="750" alt="image" src="https://github.com/user-attachments/assets/9cc9e4c3-b0c5-47de-89d6-5c44c30b08dc">

Reply:
try `SIGNUPUSERNAME`